src/sdk.ts

```diff
@@ -339,11 +332,11 @@ export class OpenSeaSDK {
     expirationTime?: BigNumberish;
     paymentTokenAddress?: string;
   }): Promise<OrderV2> {
+    //TODO: Make this function multichain compatible
```

Comment:
what happens if the fn is passed with a chain like Klaytn? should we throw an error here?

Reply:
This is calling the V1 API still so you would likely get some sort of not found exception. But we can check for Goeril or Mainnet otherwise throw an error